src/initializers/diamorphosis.ts

```diff
@@ -25,10 +25,18 @@ export default (config, orkaOptions: Partial<OrkaOptions>) => {
   config.riviere = {
     enabled: true,
     color: true,
-    styles: ['simple'],
+    styles: [],
     headersRegex: '^X-.*',
     ...config.riviere
   };
   diamorphosis(orkaOptions.diamorphosis);
   config.honeybadger.environment = config.honeybadger.environment || config.nodeEnv;
+
+  if (config.log.console === '') {
+    config.log.console = !config.log.json;
+  }
```

Comment:
if it is not empty can we make it Boolean?

Reply:
Done